src/changelog-config.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable node/no-process-env, node/no-sync */
+
+import fs from 'fs';
+import path from 'path';
+
+type PackageJson = {
+  autoChangelog: {
+    packageRename: {
+      originalLastestVersion: string;
+      originalTagPrefix: string;
+    };
+  };
+};
+
+/**
+ * Returns the latest version of the original package in case of package renamed.
+ *
+ * @returns The original package latest version.
+ */
```

Comment:
`@returns` tag should include return types for all branches.
```suggestion
 * @returns The latest version string of the original package, or null.
```